### DISCUSSION TITLE
Filter Selection Reset on click of Back link

### DISCUSSION THREAD

**Discussion by idlhardik** [bullhorn/career-portal#408] [Ideas]
I have setup career-portal on my server. User submitted an issue that after filtering selecting a job to see details of it. Once user come back to filtered list by clicking back button/link, it is not showing filtered list. Instead it is showing all jobs.

1. Got to link https://www.physiciansforyou.com/search-jobs.html and filter by category and province as shown in below screenshot.

![image](https://user-images.githubusercontent.com/81185903/112040658-dcd11580-8b1b-11eb-8447-8eacbd43472a.png)

2. Selecting first job to view details of job. 
![image](https://user-images.githubusercontent.com/81185903/112040781-fffbc500-8b1b-11eb-9046-fd8152e7fccd.png)

3. Once I click on back link/button at top-left part. I will show list without any filter that selected previously. 
![image](https://user-images.githubusercontent.com/81185903/112040988-3afdf880-8b1c-11eb-8169-78de47a36fe5.png)

Expected Result : 
On click of Back button, it should show filtered job list which was selected previously.

**Comment by charlesabarnes** [comment]
Hello, 

This is not a feature right now, but can potentially be one in a future update. For now, this is intended behavior.
